.github/workflows/database-compose-tests.yml

```diff
@@ -13,6 +13,7 @@ jobs:
     # Probably that's why the job failed on CircleCI
     if: ${{inputs.node-version != '16.x'}}
     runs-on: ubuntu-latest
+    timeout-minutes: 60
     strategy:
```

Comment:
Could you also remove `strategy`? It doesn't make sense when there's no `matrix` property.

Reply:
100% 🙏 
Great catch, done.